src/components/ExploreMovieList.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { SyntheticEvent, useState, useEffect } from "react";
+
+import {
+  Movie,
+  MoviePreference,
+  PreferenceType,
+  SetMovies,
+  User
+} from "../types";
+
+import { Card } from "semantic-ui-react";
```

Comment:
nit: it might make more sense to keep imports from external libraries separate from the imports from our own code - if we moved this below the react import I think that would work.

Reply:
Done